plugins/outputs/mongodb/mongodb.go

```diff
@@ -26,7 +26,7 @@ import (
 var sampleConfig string
 
 func (s *MongoDB) getCollections(ctx context.Context) error {
-	s.collections = map[string]bson.M{}
+	s.collections = make(map[string]bson.M)
```

Comment:
Is there a way to determine the size of the map before to prealloc it?

Reply:
We can try.